apps/etterlatte-brev-api/src/main/kotlin/no/nav/etterlatte/rivers/migrering/FiksEnkeltbrevRiver.kt

```diff
@@ -42,23 +35,7 @@ internal class FiksEnkeltbrevRiver(
         logger.info("Fikser vedtaksbrev for behandling $behandlingId")
 
         val brukerTokenInfo = Systembruker("migrering", "migrering")
-        val sum = packet[SUM].asInt()
-        val utlandstilknytningType = packet[UTLANDSTILKNYTNINGTYPE].asText().let { UtlandstilknytningType.valueOf(it) }
-        val migreringBrevRequest =
-            MigreringBrevRequest(brutto = sum, yrkesskade = false, utlandstilknytningType = utlandstilknytningType)
         runBlocking {
-            val sakId = vedtaksvurderingService.hentVedtak(behandlingId, brukerTokenInfo).sak.id
-            val vedtaksbrev: Brev =
-                service.opprettVedtaksbrev(
-                    sakId,
-                    behandlingId,
-                    brukerTokenInfo,
-                    migreringBrevRequest,
-                )
-            service.genererPdf(vedtaksbrev.id, brukerTokenInfo, migreringBrevRequest)
-            service.ferdigstillVedtaksbrev(behandlingId, brukerTokenInfo, true)
-            logger.info("Har oppretta vedtaksbrev for behandling $behandlingId")
-
             packet.eventName = VedtakKafkaHendelseType.ATTESTERT.toString()
```

Comment:
Vil vi nå forsøke å iverksette mot oppdrag på nytt også?

Reply:
VedtakMottakRiver har no `validate { it.rejectKey(FIKS_BREV_MIGRERING) }`, som er ein key vi definerer i meldinga vi lager i ApplicationBuilder her.

Klok av skade etter ei tidlegare køyring, som eg hugsar det.